webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/controllers/VolumeTracingController.scala

```diff
@@ -164,7 +164,7 @@ class VolumeTracingController @Inject()(
         accessTokenService.validateAccess(UserAccessRequest.readTracing(tracingId), urlOrHeaderToken(token, request)) {
           for {
             tracing <- tracingService.find(tracingId) ?~> Messages("tracing.notFound")
-            (data, indices) <- if (tracing.mappingIsEditable.getOrElse(false))
+            (data, indices) <- if (tracing.hasEditableMapping.getOrElse(false))
```

Comment:
`getHasEditableMapping` could be used here afaik
```suggestion
            (data, indices) <- if (tracing.getHasEditableMapping)
```